### PR TITLE
fix: remove loading default font

### PR DESCRIPTION
**Pull request by aeksandla**

### CODE REVIEW

**Comment by aeksandla** [comment]
breaking changes

**Comment by NikitaCG** [comment]
> breaking changes

change target branch on next

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gravity-ui/page-constructor",
-  "version": "5.30.0",
+  "version": "5.30.0-alpha.0",
```
Comment (NikitaCG) [inline package.json:3]: fix version

File: README.md [hunk 2]
```diff
@@ -165,6 +165,12 @@ To use mixins and constructor style variables when creating custom blocks, add i
 @import '~@gravity-ui/page-constructor/styles/styles.scss';
 ```
 
+To use default font, add import in your file:
+
+```css
+@import '@gravity-ui/uikit/styles/fonts.css';
```
[the remark this answers is not in the document]
Comment (NikitaCG) [reply README.md:171]: yes)

**Review by NikitaCG — APPROVED** [review]

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (aeksandla) [reply README.md]: nice idea

**Pull request merged**